credoai/evaluators/shap.py

```diff
@@ -14,15 +16,16 @@ class ShapExplainer(Evaluator):
     This evaluator perform the calculation of shapley values for a dataset/model,
     leveraging the SHAP package.
 
-    It calculates 2 types of assessments:
+    It supports 2 types of assessments:
     1. Overall statistics of the shap values across all samples: mean and mean(|x|)
-    2. Optional: individual shapley values for a list of samples
+    2. Individual shapley values for a list of samples
 
     Parameters
     ----------
     samples_ind : Optional[List[int]], optional
         List of row numbers representing the samples for which to extract individual
-        shapley values , by default None
+        shapley values. This must be a list of integer indices. The underlying SHAP
+        library does not support non-integer indexing
     background_kmeans : bool, optional
         If True, use SHAP kmeans to create a data summary to serve as background data for the
```

Comment:
Add a comment saying we are doing this because SHAP is slow (maybe even link to Scott Lundberg's advice on this from Github posts we linked in our discussion).